### ISSUE TITLE
Readme installation instructions give error as Git clone does not automatically fetch submodules

### ISSUE THREAD

**Issue by noahsprent** [Bioprotocols/paml#116]
Following along with the Installation instructions in the README will lead users to clone the PAML repo but they will recieve an error at this stage:

```
cd paml/paml-check
pip3 install .
ERROR: Directory '.' is not installable. Neither 'setup.py' nor 'pyproject.toml' found.
```

Because by default git clone doesn't clone submodules, so the paml-check directory is empty ([see here](https://stackoverflow.com/questions/3796927/how-to-git-clone-including-submodules)). The --recurse-submodules option should be added to ensure this.

**Comment by danbryce** [comment]
This could be due to how we moved pail-check (and some others) into git submodules.  I believe you don’t have anything there because you need to run (in the submodule directory):

```
git submodule init
git submodule update
```

See also: https://git-scm.com/book/en/v2/Git-Tools-Submodules

**Comment by noahsprent** [comment]
Yes, that is exactly the issue. I think either those instructions or the --recurse-submodules option should be added to the README installation section.
